packages/eas-cli/src/project/remoteVersionSource.ts

```diff
@@ -98,3 +134,96 @@ export function getBuildVersionName(platform: Platform): string {
     return 'buildNumber';
   }
 }
+
+export async function ensureAppVersionSourceIsSetAsync(
+  easJsonAccessor: EasJsonAccessor,
+  easJsonCliConfig: EasJson['cli'] | undefined,
+  nonInteractive: boolean
+): Promise<EasJson['cli'] | undefined> {
+  let selectOption, updateEasJson;
+  if (nonInteractive) {
+    Log.warn(
+      `The field "cli.appVersionSource" is not set, but it will be required in the future. ${learnMore(
```

Comment:
```suggestion
    Log.warn(
      `The field "cli.appVersionSource" is not set, but it will be required in the future Proceeding with the default "local" value. ${learnMore(
```